environment

```diff
@@ -333,19 +333,33 @@ _hibernate() {
 		make -C lambdas && {
 			cd terraform &&
 			make validate
```

Comment:
```suggestion
			make validate && {
```

The addition of `&&` fixes a preexisting bug, so separate commit. The addition of `{` should be in the main commit.